Assets/Scripts/BoulderController.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+
+/// <summary>
+/// Handles the physics and features behind the boulders 
+/// </summary>
+public class BoulderController : MonoBehaviour
+{
+    private Rigidbody2D rb;
```

Comment:
I'm getting warnings about a few things here too- file should ideally have no warnings in Rider